### PR TITLE
Remove extRefresh cmd & FTP registration popup

### PR DESCRIPTION
**Pull request by JillieBeanSim**
## Proposed changes

fix for #2862 & #3100 

log writing can be found at `~/.vscode/extension/zowe.zowe-explorer-ftp-extension-3.0.0-next-SNAPSHOT/logs`

## Release Notes

Milestone: 3.0 GA

Changelog:

ZE - Removal of broken VSC command `Zowe Explorer: Refresh Zowe Explorer`, use VS Code's `Extensions: Refresh` command instead. [#3100](https://github.com/zowe/zowe-explorer-vscode/issues/3100)
zFTP - Removed popup notification of successful registration with Zowe Explorer and log the success instead. [#2862](https://github.com/zowe/zowe-explorer-vscode/issues/2862)

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Enhancement (non-breaking change which adds or improves functionality)
- [ ] Breaking change (a change that would cause existing functionality to not work as expected)
- [ ] Documentation (Markdown, README updates)
- [ ] Other (please specify above in "Proposed changes" section)

## Checklist

**General**

- [x] I have read the [CONTRIBUTOR GUIDANCE](https://github.com/zowe/zowe-explorer-vscode/wiki/Best-Practices:-Contributor-Guidance) wiki
- [ ] All PR dependencies have been merged and published (if applicable)
- [ ] A GIF or screenshot is included in the PR for visual changes
- [x] The pre-publish command has been executed:
  - **v2 and below:** `yarn workspace vscode-extension-for-zowe vscode:prepublish`
  - **v3:** `pnpm --filter vscode-extension-for-zowe vscode:prepublish`

**Code coverage**

- [ ] There is coverage for the code that I have added
- [ ] I have added new test cases and they are passing
- [ ] I have manually tested the changes

**Deployment**

- [ ] I have added developer documentation (if applicable)
- [ ] Documentation should be added to Zowe Docs
  - If you're an outside contributor, please post in the [#zowe-doc Slack channel](https://openmainframeproject.slack.com/archives/CC961JYMQ) to coordinate documentation.
  - Otherwise, please check with the rest of the squad about any needed documentation before merging.
- [ ] These changes may need ported to the appropriate branches (list here):

### CODE REVIEW

**Review by zFernand0 — APPROVED** [review]

**Comment by JillieBeanSim** [comment]
@zFernand0 I presented the proposal with the v2 [fix](https://github.com/zowe/zowe-explorer-vscode/issues/3100) and it was discussed in scrum to remove the command in favor of the VSC command.

File: packages/zowe-explorer/CHANGELOG.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ All notable changes to the "vscode-extension-for-zowe" extension will be documen
 - Fix issue with extender profiles not being included in fresh team configuration file. [#3122](https://github.com/zowe/zowe-explorer-vscode/issues/3122)
 - Fixed issue where file extensions were removed from data sets, causing language detection to sometimes fail for Zowe Explorer extenders. [#3121](https://github.com/zowe/zowe-explorer-vscode/issues/3121)
 - Fixed an issue where copying and pasting a file/folder in the USS tree would fail abruptly, displaying an error. [#3128](https://github.com/zowe/zowe-explorer-vscode/issues/3128)
-- To fix Strange behaviour of Job label under Job Favorites. [#2632](https://github.com/zowe/zowe-explorer-vscode/issues/2632)
```
Comment (JillieBeanSim) [inline packages/zowe-explorer/CHANGELOG.md:17]: removed, v2 port changelog

**Review by zFernand0 — APPROVED** [review]

File: packages/zowe-explorer-ftp-extension/src/extension.ts [hunk 2]
```diff
@@ -44,7 +44,7 @@ async function registerFtpApis(): Promise<boolean> {
         await zoweExplorerApi.getExplorerExtenderApi().initForZowe(pType, schema);
         await zoweExplorerApi.getExplorerExtenderApi().reloadProfiles(pType);
 
-        await Gui.showMessage("Zowe Explorer was modified for FTP support.", { logger: globals.LOGGER });
+        globals.LOGGER.logImperativeMessage("Zowe Explorer was modified for FTP support.", MessageSeverity.INFO);
```
Comment (traeok) [inline packages/zowe-explorer-ftp-extension/src/extension.ts:47]: Thanks for removing this! 😋

**Review by traeok — APPROVED** [review]
LGTM, thanks Billie for resolving these clean-up items!

**Pull request merged**